### PR TITLE
Add capvcd architecture doc

### PR DESCRIPTION
**Pull request by vxav**
### What does this PR do?

Adds the CAPVCD section to CAPI architecture.

### What does it look like?

(If this is more than a textual change, a screenshot can help in some cases to speed up the review process.)

### Any background context you can provide?

(Please link public issues or summarize if not public.)

### Have you maintained the front matter?

(Please bump the last_review_date in case this qualifies as a review for an entire page. Provide user_questions which should be answered in the page. Provide a meaningful description.)

### CODE REVIEW

File: src/content/platform-overview/cluster-management/cluster-api/architecture/index.md [hunk 1]
```diff
@@ -36,6 +36,31 @@ There is a machine created as a bastion that helps us with the operations. It is
...
+
+### Networking
+
+The kubernetes API and services type `LoadBalancer` get IPs from the pool of external IPs available in the edge gateway, it can be set statically or it will take the next available IP. A virtual service is then created with the selected IP/port and is associated with a load balancer pool that contains the relevant node IPs as members. For the CPI, we support the virtual service shared feature introduced in VCD 10.4 as well as the legacy method based on a single internal IP and multiple DNAT rules.
+
+To connect the nodes, a network needs to be specified in CAPVCD to define where the default gateway will be. It is also possible to add additional networks in order to connect multiple virtual interfaces to the nodes along with the configuration of static routes. The nodes must have internet access which usually achieved with an SNAT rule or via an HTTP proxy. Note that it is possible to specify NTP servers and pools (Ubuntu based nodes running `chrony`) which is particularly useful in air-gapped environments.
```
Comment (pipo02mix) [inline src/content/platform-overview/cluster-management/cluster-api/architecture/index.md:54]:
```suggestion
To connect the nodes, a network needs to be specified in CAPVCD to define where the default gateway will be. It is also possible to add additional networks in order to connect multiple virtual interfaces to the nodes along with the configuration of static routes. The nodes must have internet access which is usually achieved with an SNAT rule or via an HTTP proxy. Note that it is possible to specify NTP servers and pools (Ubuntu based nodes running `chrony`) which is particularly useful in air-gapped environments.
```

**Comment by pipo02mix** [comment]
One minor thing, can you add the entire word the first time an acronym is used? like CSI -> Container Storage Interface (CSI), and then you can use just the acronym. That helps ppl no so knowledgable 

I think it is a good summary but looking at the rendered style I think we need to work on the format. I can do a PR against your if you don't mind
Screenshot 2023-06-08 at 11 16 19

File: src/content/platform-overview/cluster-management/cluster-api/architecture/index.md [hunk 2]
```diff
@@ -36,6 +36,31 @@ There is a machine created as a bastion that helps us with the operations. It is
 
 The setup requires an external network configured in the project to allow the machines to pull images or route requests from containers to the Internet. On the other side, there is an internal network which interconnects all master and worker machines allowing the internal communication of all containers within the cluster. At the same time, it allocates the load balancers that are created dynamically as result of exposing a service in the cluster. In case the load balancer is external, a floating IP is allocated to enable the connection with external endpoints.
 
+{{< /tab >}}
+{{< tab id="flags-clouddirector" title="CAPVCD (VMware Cloud Director)">}}
```
[the remark this answers is not in the document]
Comment (pipo02mix) [reply src/content/platform-overview/cluster-management/cluster-api/architecture/index.md:40]: yeah just provider name

[the remark this answers is not in the document]
Comment (vxav) [reply src/content/platform-overview/cluster-management/cluster-api/architecture/index.md]: Ok done. I'm marking this PR as ready.

**Review by pipo02mix — APPROVED** [review]
noice

**Pull request merged**